src/components/ResultTable.js

```diff
@@ -9,15 +9,33 @@ function calculateOverallStrength(cells) {
 }
 
 class ResultTable extends React.Component {
+	constructor(props) {
+		super(props);
+	}
 	render() {
 		const { data } = this.props;
 		return (
 			<table cellSpacing={8}>
 				<thead>
 					<tr>
-						<th>Overall</th>
-						<th>Organ</th>
-						<th>Cells</th>
+						<th>
+							<span className="overall" onClick={this.props.handleClick}>
+								▲
```

Comment:
These characters don't look too good to me, can we instead use chevron icons?
You are free to use `react-feather-icons` library, for reference for chevron from feather-icons: https://feathericons.com/?query=chevron

Reply:
I just thought it would lower page loading time than icons, found that icons usually add 75k to 100k extra page weight or you don't like the design?

Reply:
Actually, if you use those two icons, they'll only take approx 10-25 kb extra only.